library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
+	parameter x,y,w,h.
+	// w and h are INTERNAL width and height
```

Comment:
The program might want to print its title on the border above or copyright/footnote below. And the border itself is what I want to be common in all window programs, so making each program draw the border itself is probably not really what we need.